docs_src/general/ContainerNames.md

```diff
@@ -50,4 +50,4 @@ The following table provides the list of the default EdgeX Docker image names to
     |Docker image name|Docker container name |Docker network hostname|Docker Compose service name|
     |---|---|---|---|
     |consul|edgex-core-consul|edgex-core-consul|consul|
```

Comment:
remove the rwo of consul here and replace vault with secret-store and openbao in line 43